### PR TITLE
merging template updates v3.0.2

### PR DESCRIPTION
**Pull request by d4straub**
Template update 3.0.2

Testing with conda is excluded, because several profiles do not work with conda. It would be interesting though if there is a nice way to run some test profiles with conda and some without.

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/ampliseq/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/ampliseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by d4straub** [comment]
@nf-core-bot fix linting

**Comment by d4straub** [comment]
`test` fails because the reference taxonomy database via `dada_ref_taxonomy = "gtdb=R07-RS207"` seems to be not reachable at the moment, i.e. https://data.ace.uq.edu.au/public/gtdb/data/releases/ doesnt work.
Will wait a little. Other options might be  `sbdi-gtdb=R06-RS202-3` which is at https://scilifelab.figshare.com or `rdp=18` that links to zenodo.

**Comment by erikrikarddaniel** [comment]
Earlier, we had some problems with the main site for GTDB that I think are solved now. You could update the config to use paths here: https://data.gtdb.ecogenomic.org/releases/release207/207.0/

**Comment by d4straub** [comment]
Great thanks, trying that out!

File: CITATIONS.md
[the remark this answers is not in the document]
Comment (d4straub) [reply CITATIONS.md:1]: Thanks, I'll go back to the original indentation!

**Pull request merged**